datahub-web-react/src/app/entity/shared/containers/profile/header/EntityName.tsx

```diff
@@ -27,6 +29,7 @@ function EntityName(props: Props) {
     const { isNameEditable } = props;
     const refetch = useRefetch();
     const entityRegistry = useEntityRegistry();
+    const { isInGlossaryContext, updatedUrns, setUpdatedUrns } = useGlossaryEntityData();
```

Comment:
This is a bit unfortunate but I see why we have to do it. To communicate this information...

Reply:
yeah :/ just the tough thing with having the same info displayed in separate components with different fetching happening in different places.